app/src/pages/Play/TurnContextPanel.tsx

```diff
@@ -26,19 +28,29 @@ function CardsLeftItem() {
 
 function ScoreCardItem() {
   const currentGame = React.useContext(CurrentGameContext)
+  const numTurns = currentGame.turns.length
+  const [fetchStats, { data }] = useGameStatsLazyQuery({
+    fetchPolicy: "no-cache",
+  })
+
+  React.useEffect(() => {
+    fetchStats()
+  }, [numTurns, fetchStats])
```

Comment:
the best I could come up with to keep this score up to date (only runs, when the number of turns change -- i.e. when a new turn is created as part of someone's turn finishing)

no-cache hits server directly, and doesn't cache the result - https://medium.com/@galen.corey/understanding-apollo-fetch-policies-705b5ad71980